src/adapters/SwivelAdapter.sol

```diff
@@ -46,15 +47,15 @@ contract SwivelAdapter is IAdapter, Lender {
             );
 
         // Cache a couple oft-referenced variables
-        address underlying = orders[0].underlying;
+        address underlying_ = orders[0].underlying;
         uint256 maturity = orders[0].maturity;
-        address pt = IMarketPlace(marketPlace).markets(underlying, maturity, 1);
+        address pt = IMarketPlace(marketPlace).markets(underlying_, maturity).tokens[1];
 
-        // verify orders are for the same underlying
+        // Verify orders are for the same underlying
         {
             for (uint256 i = 0; i < orders.length; ) {
                 if (
-                    underlying != orders[i].underlying ||
+                    underlying_ != orders[i].underlying ||
```

Comment:
just `underlying`?

Reply:
I think both work actually -- one is directly read from the PT, while one is part of the mapping that points to the PT -- I just wanted to be consistent with the method as im using `underlying_` elsewhere in the method

Reply:
👍